settings.gradle.kts

```diff
@@ -28,7 +31,7 @@ include(":examples:android-gradle")
 
 include(":examples:android-gradle-kts")
 
-include(":examples:android-guardsquare-proguard")
+// include(":examples:android-guardsquare-proguard")
```

Comment:
For some reason this sample doesn't work but I will figure it out later (or maybe drop the sample altogether, as it's problematic to support it given there's [no plans](https://github.com/Guardsquare/proguard/issues/280) to support AGP 8+ from the proguard side)